defaults/linuxrc

```diff
@@ -601,16 +601,16 @@ do
 						for i in ${BOOTFS}
 						do
 
-							zfs get type ${i} > /dev/null
+							zfs get type ${i} > /dev/null 2>&1
```

Comment:
`2>&1` should just replace `>` probably

Reply:
I'm not sure what you mean, some short form?
it discards both stdout and stderr in a canonical way.

just checking return type.
without redirecting stderr it leaks in some cases while guessing rootfs. and we certainly don't want to see standard  output here and not using it at all.

Reply:
ah, makes sense